tests/nbs/phase_zero.ipynb

```diff
@@ -0,0 +1,320 @@
+{
```

Comment:
please do a pre-commit install when working with this repository to prevent notebooks outputs to be committed :)

Reply:
It is installed now :)